docs/acceptance-criteria/uacs-tbtc-v2.adoc

```diff
@@ -0,0 +1,291 @@
+  - are there no wallets or do all wallets have their balances higher or equal
+  `wallet_min_btc`? (if not, then notification invalid).
+
+| Wallet lifecycle - wallet creation
+| If the time-to-create-new-wallet notification is deemed valid, then: +
+  - the notifier should receive reimbursement for the gas spent on the
+  transaction, paid from the DAO-funded ETH pool to the notifier’s address (in
+  the same tx); there should be a limit on the refunded gas price
+  (`max_gas_refund_price`) (if the amount of gwei paid for gas in the DKG result
+  submission is bigger than this value, the submitter will be reimbursed
+  `max_gas_refund_price x gas_used`) +
+  - the wallet registry contract should lock the sortition pool and request a
+  new relay entry from the Random Beacon +
+  - once the value is returned, the wallet registry should start a process of
+  distributed key generation (DKG).
+
+| Wallet lifecycle - wallet creation - DKG
+| At the beginning of the DKG process the DKG contract should emit an event
+  informing about the start of the DKG and the value of current relay entry.
+
+| Wallet lifecycle - wallet creation - DKG - selecting members
+| Upon noticing the DKG start event, all off-chain ECDSA clients should call the
+  sortition function which should select `group_size` (100) pool members to the
+  candidate group based on the current relay entry, ensuring that the higher is
+  the stake of an operator, the higher is his chance of being selected to the
+  group (operators may be chosen to the group multiple times).
+
+// It's not yet clear if any member will be able to submit the result or
+// if there will be some order enforced. Following UAC assumes "any".
+| Wallet lifecycle - wallet creation - DKG
+| After group members are determined, clients should perform off-chain DKG
+  (distributed key generation), resulting in either a success or a timeout. To
+  constitute a success at least `dkg` (90) members of the group must actively
+  participate in the DKG process. The success result should contain a list of
+  members of the candidate group, the public key of the group and a list of
+  misbehaving members (members who were inactive or were disqualified) and
+  should be submitted by any member of the candidate group on-chain.
+
+// TODO: provide more details once we have that figured out; we can remove this
+// UAC if the order of result submission will be somehow enforced.
+| Wallet lifecycle - wallet creation - DKG submission
+| There should be some mechanism implemented which would discourage operators
+  from front-running each other (and bumping up the fee) when submitting DKG
+  result.
+
+| Wallet lifecycle - wallet creation - DKG submission
+| When determining the validity of the transaction submitting the DKG result,
+  the DKG contract must check if the submitter of the result is a member of the
+  sortition pool. If it is, the transaction should be considered valid.
```

Comment:
Don't we need to also check if signatures come from stakers with some minimum stake?